### PR TITLE
implement env var based node and file limits on cortex and axon

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/axon.py [hunk 1]
```diff
@@ -242,13 +241,39 @@ async def __anit__(self, dirn, conf=None):  # type: ignore
...
+            self.maxcount <= self.axonmetrics.get('file:count')):
+            mesg = f'Axon is at file:count limit: {self.maxcount}'
+            raise s_exc.HitLimit(mesg=mesg)
+
+    def _initAxonLimits(self):
+
+        self.maxbytes = None
+        self.maxcount = None
+
+        maxcount = os.getenv('AXON_MAX_COUNT')
```
Comment (vEpiphyte) [inline synapse/axon.py:269]: Why not bake this into the cellconf ?

Comment (invisig0th) [reply synapse/axon.py:269]: Mostly because we dont want it to be very visible, but if you think it should be i guess we could.

Comment (vEpiphyte) [reply synapse/axon.py:269]: per discussionn, going to plumb this in as a hidden conf option.

File: synapse/axon.py [hunk 2]
```diff
@@ -242,13 +255,39 @@ async def __anit__(self, dirn, conf=None):  # type: ignore
...
+            raise s_exc.HitLimit(mesg=mesg)
+
+        if (self.maxcount is not None and
+            self.maxcount <= self.axonmetrics.get('file:count')):
+            mesg = f'Axon is at file:count limit: {self.maxcount}'
+            raise s_exc.HitLimit(mesg=mesg)
+
+    def _initAxonLimits(self):
+
+        maxcount = self.conf.get('max:count')
```
Comment (invisig0th) [inline synapse/axon.py:283]: Dont think we need this conditional paradigm anymore.  can just `self.maxcount = self.conf.get`

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**